src/structured-text/converters/convert_string_value.cpp

```diff
@@ -0,0 +1,43 @@
+/*******************************************************************\
+
+Module: Structured Text Language Conversion
+
+Author: Matthias Weiss, matthias.weiss@diffblue.com
+
+\*******************************************************************/
+
+/// \file
+/// Structured Text Language Conversion
+
+#include "convert_string_value.h"
+
+exprt convert_identifier(const std::string &src)
```

Comment:
Both `convert_identifier` and `convert_title` can probably just return `string_constantt`.